### PR TITLE
Store schema chagnes

### PR DESCRIPTION
**Pull request by jsnoble**
- fix s3 schema in terafoundation
- change default kafka version for testing to 3.1

### CODE REVIEW

File: packages/terafoundation/src/connectors/s3.ts [hunk 1]
```diff
@@ -50,12 +50,12 @@ export default {
...
                 format: String
             },
-            s3ForcePathStyle: {
+            forcePathStyle: {
                 doc: '',
                 default: false,
                 format: Boolean
             },
-            s3BucketEndpoint: {
+            bucketEndpoint: {
```
Comment (godber) [inline packages/terafoundation/src/connectors/s3.ts:58]: What is the difference between `endpoint` and `bucketEndpoint`?

**Review by godber — COMMENTED** [review]
The rest of this PR looks good, I'm just concerned that we're overlooking something WRT `endpoint` and `bucketEndpoint`.  How is each of those used?

File: packages/terafoundation/src/connectors/s3.ts (hunk 1 above)
Comment (jsnoble) [reply packages/terafoundation/src/connectors/s3.ts:58]: you can check the [migration](https://github.com/aws/aws-sdk-js-v3/blob/main/UPGRADING.md) here and see about forcePathStyle and see what bucketEndpoint does

**Review by godber — APPROVED** [review]
Approved, thanks Jared!

**Pull request merged**